app/src/test/java/com/example/trainingintive/viewmodels/SplashViewModelTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.trainingintive.viewmodels
+
+import com.example.trainingintive.TestSchedulersProvider
+import com.example.trainingintive.navigators.SplashNavigator
+import com.example.trainingintive.util.SplashScreenEvent
+import io.mockk.justRun
+import io.mockk.mockk
+import io.mockk.verify
+import io.reactivex.rxjava3.plugins.RxJavaPlugins
+import io.reactivex.rxjava3.schedulers.TestScheduler
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.util.concurrent.TimeUnit
+
+class SplashViewModelTest {
+
+    private lateinit var viewModel: SplashViewModel
+    private val navigator = mockk<SplashNavigator> {
+        justRun { sendEvent(any()) }
+    }
+    private val schedulersForTest = TestSchedulersProvider()
+
+    @Before
+    fun setUp() = RxJavaPlugins.reset()
+
+    @After
+    fun tearDown() = RxJavaPlugins.reset()
```

Comment:
Why do you reset this before and after every test? I think you should do it before or after and it will be enought.